internal/sync/utils.go

```diff
@@ -64,6 +64,7 @@ func loadAbiOfContract(abiVal string) (abi.ABI, error) {
 func fetchContractCode(stateDiff *types.StateDiff, client *feeder.Client, logger *zap.SugaredLogger) *CollectorDiff {
 	collectedDiff := &CollectorDiff{
 		stateDiff: stateDiff,
+		Code:      map[string]*types.Contract{},
```

Comment:
Is it possible to use `make` here and set the map size properly with `len` of `stateDiff.DeployedContracts`?

Reply:
Changed to make, good catch!